mozdownload/parser.py

```diff
@@ -56,6 +56,7 @@ def handle_data(self, data):
         if not self.active_url:
             return
 
-        name = urllib.quote(data.strip('/'))
```

Comment:
Try entries have the form `user@domain-<changeset>`. It appears some servers escape the `@` (mozhttpd which our unit tests use does) and some don't (http://ftp.mozilla.org/pub/mozilla.org/firefox/try-builds/).